spec/components/collections/information_component_spec.rb

```diff
@@ -0,0 +1,23 @@
+# typed: false
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Collections::InformationComponent, type: :component do
+  let(:rendered) { render_inline(described_class.new(collection: collection)) }
+
+  context 'when displaying a collection' do
+    let(:created) { I18n.l(collection.created_at, format: :abbr_month) }
+    let(:last_saved) { I18n.l(collection.updated_at, format: :abbr_month) }
+    let(:version) { collection.version.to_s }
+    let(:creator) { collection.creator.to_s }
+    let(:collection) { build_stubbed(:collection) }
+
+    it 'renders the information component' do
+      expect(rendered.css('table').to_html).to include(version)
+      expect(rendered.css('table').to_html).to include(creator)
+      expect(rendered.css('table').to_html).to include(created)
+      expect(rendered.css('table').to_html).to include(last_saved)
```

Comment:
I'm not sure these are good tests.  You test is running the same code as the code being tested is running. Effectively you are asking does `x = x`?